.github/workflows/release.yml

```diff
@@ -84,20 +86,24 @@ jobs:
               generate_release_notes: true
             })
 
-      - name: Check commit messages for feat or break
+      - name: Check commit messages and descriptions for feat or break
         id: check_commits
         run: |
           PUBLISH_DOCS=false
+          COMMIT_DETAILS=""
           for COMMIT in $(git rev-list ${{ github.event.before }}..${{ github.sha }}); do
```

Comment:
The COMMIT_DETAILS variable should be initialized before the loop to accumulate commit details properly.